src/org/labkey/test/WebTestHelper.java

```diff
@@ -389,27 +394,26 @@ public static String buildRelativeUrl(String controller, @Nullable String contai
 
     public static Connection getRemoteApiConnection()
     {
-        return getRemoteApiConnection(false);
+        return getRemoteApiConnection(true);
```

Comment:
I want to encourage tests to use a `Connection` tied to the current browser session unless they have specific reasons not to. Most scenarios don't need a totally fresh `Connection` and this will make tests more likely to work against CAS-only servers.